packages/api-rpc/src/api/update_auth.ts

```diff
@@ -1,6 +1,6 @@
 import { Api, JsonRpc } from 'eosjs';
 import { JsSignatureProvider } from 'eosjs/dist/eosjs-jssig';
-import { TextDecoder, TextEncoder } from 'text-encoding';
```

Comment:
is it a must to make this change?
I forget the reason but it is quite important that we add `TextDecoder, TextEncoder` on the time being

Reply:
I replace `text-encoding` with another package `fast-text-encoding`, because `text-encoding` is deprecated now.  `fast-text-encoding` only require to be imported once and I did it in `packages/api-rpc/src/index.ts`